File: Second Year (2013-2014)/OMGL7/TP2 - Website in PHP/musiconline/musiconline/db/Pays.php
<?php
/* Authors: ANDRIAMILANTO Tompoariniaina && CANNO Camille
* Groupe: B1
* File: Pays.php
*/

class Pays implements DataObject{
   private $IdPays;
   private $Nom;

   public function  __construct() { }

   public function init($IdPays,$Nom){
      $this->IdPays=$IdPays;
      $this->Nom=$Nom;
   }


   public function getIDPays(){
     return $this->IdPays;
   }

   public function setIDPays($IdPays){
     $this->IdPays = $IdPays;
   }


   public function getNom(){
     return $this->Nom;
   }

   public function setNOm($Nom){
     $this->Nom = $Nom;
   }


}
?>

File: Second Year (2013-2014)/OMGL7/TP3 - Website in JSP/src/db/Pays.java
package db;
/**
* @author Andriamilanto Tompoariniaina - Canno Camille - Info2B
*/

/**
* Classe pour les objets Pays
*/
public class Pays{
//Attributs
	private int idPays;
	private String nom;
	 
	/**
	*Constructeur de la classe Pays
	*/
	public Pays(){}
	  
	/**
	* Méthode qui initialise les attributs de l'objet Pays
	*/
	public void init(int idp, String name){
		this.idPays = idp;
		this.nom = name;
	}
	  
	/**
	* Accesseur de l'objet
	* @return IdPays
	*/
	public int getIdPays(){
		return this.idPays;
	}
	   
	/**
	* Modificateur de l'objet
	*/
	public void setIdPays(int idp){
		this.idPays = idp;
	}
	   
	/**
	* Accesseur de l'objet
	* @return nom
	*/
	public String getNom(){
		return this.nom;
	}
		
	/**
	* Modificateur de l'objet
	*/
	public void setNom(String name){
		this.nom = name;
	}
}